client/layout/universal-navbar-footer/index.tsx

```diff
@@ -305,7 +305,7 @@ const UniversalNavbarFooter = () => {
 													{ translate( 'Download on the' ) }
 												</span>
 												<span className="lp-app-button__line lp-app-button__line--bottom">
-													{ translate( 'App Store' ) }
+													App Store
```

Comment:
"App Store" should not be localized: https://developer.apple.com/app-store/marketing/guidelines/